Stack/Opc.Ua.Core/Stack/Tcp/UaSCBinaryChannel.Asymmetric.cs

```diff
@@ -714,7 +866,7 @@ private int GetMaxSenderCertificateSize(X509Certificate2 senderCertificate, stri
         [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Performance", "CA1804:RemoveUnusedLocals", MessageId = "messageType"), System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Performance", "CA1804:RemoveUnusedLocals", MessageId = "messageSize")]
         protected void ReadAsymmetricMessageHeader(
             BinaryDecoder decoder,
-            X509Certificate2 receiverCertificate,
+            ref X509Certificate2 receiverCertificate,
```

Comment:
how come? Is the receiver cert loaded due to the response? Shouldn't it be loaded before based on the sec policy or endpoint?